### PR TITLE
Pinterest Ads - Added LDP support

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updated Pinterest Ads cloud mode doc to add section on LDP support.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 1]
```diff
@@ -182,6 +182,20 @@ The following table lists the custom fields mappings for `track` and `page` call
 | `properties.order_id` | `order_id` | String|
 | `properties.query` | `search_string` | String |
 
+## Limited Data Processing (LDP)
+
+Starting January 1, 2023, you can use the Limited Data Processing (LDP) flag to limit how Pinterest uses certain data to help the advertisers comply with the users' privacy settings in accordance with the CCPA (California Consumer Privacy Act).
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:187]:
```suggestion
Starting January 1, 2023, you can use the [Limited Data Processing (LDP)](https://developers.pinterest.com/docs/conversions/conversion-management/#Understanding%20Limited%20Data%20Processing) flag to limit how Pinterest uses certain data to help advertisers comply with the users' privacy settings in accordance with the CCPA (California Consumer Privacy Act).
```

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 2]
```diff
@@ -182,6 +182,20 @@ The following table lists the custom fields mappings for `track` and `page` call
...
+
+To read more about this feature, refer to the [Pinterest documentation](https://developers.pinterest.com/docs/conversions/conversion-management/#Understanding%20Limited%20Data%20Processing).
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:189]: Can remove this if including the link above looks okay

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 3]
```diff
@@ -182,6 +182,20 @@ The following table lists the custom fields mappings for `track` and `page` call
...
+
+The following table lists the event properties required to enable LDP and their mappings with the Pinterest fields:
+
+| RudderStack property | Pinterest property | Data type | Description |
+| :----------------------| :------------------| :----------| :-----------|
+| `properties.optOutType` |  `custom_data.opt_out_type` | String | Set this field to `LDP`. |
+| `traits.address.state` <br /> `context.traits.address.state` | `st` | Array of strings with SHA-256 encoding | Should be a two-letter code. |
+|  `traits.address.country`/`context.traits.address.country` | `country` | Array of strings with SHA-256 encoding |  Should be a two-character ISO-3166 country code. |
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:197]:
```suggestion
|  `traits.address.country` <br /> `context.traits.address.country` | `country` | Array of strings with SHA-256 encoding |  Should be a two-character ISO-3166 country code. |
```

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 4]
```diff
@@ -182,6 +182,20 @@ The following table lists the custom fields mappings for `track` and `page` call
 | `properties.order_id` | `order_id` | String|
 | `properties.query` | `search_string` | String |
 
+## Limited Data Processing (LDP)
+
+Starting January 1, 2023, you can use the Limited Data Processing (LDP) flag to limit how Pinterest uses certain data to help the advertisers comply with the users' privacy settings in accordance with the CCPA (California Consumer Privacy Act).
+
+To read more about this feature, refer to the [Pinterest documentation](https://developers.pinterest.com/docs/conversions/conversion-management/#Understanding%20Limited%20Data%20Processing).
+
+The following table lists the event properties required to enable LDP and their mappings with the Pinterest fields:
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:191]: IIUC, all the below properties are **required** for LDP. We can either:
* Mark all of them as required in the below table, OR
* Format "required" as bold in this line

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:191]: Let's do the latter, as the first option will then require us to update the entire doc with `Required` fields, for consistency. We can do that as a separate doc update task later.

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:189]: Agreed!

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx (hunk 4 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:191]: On second thoughts, marked as required and made changes throughout :)

**Review by ishachopra0401 — APPROVED** [review]

**Review by Gauravudia — APPROVED** [review]

**Pull request merged**